docs/Solidity.g4

```diff
@@ -382,8 +382,7 @@ HexDigits
   : HexCharacter ( '_'? HexCharacter )* ;
 
 NumberUnit
-  : 'wei' | 'szabo' | 'finney' | 'ether'
```

Comment:
I think we need to keep them for backwards compatibility in the grammar?

Reply:
Since when does the grammar need to be backwards compatible?

Reply:
Apparently the grammar is permissive so that it can be used on all versions of Solidity.